### PR TITLE
Fix Tamper Machine crash and timings

### PR DESCRIPTION
**Pull request by Caian**
Discord user mageven reported two issues with the Tamper Machine that are addressed in this PR:

- The first one is a crash when using unpacked games, caused by a missing `return` statement.
- The second one was an inconsistency between the sleep duration in Tamper Machine (1ms) versus Atmosphere (~83.33ms). Time sensitive cheats (if any) may behave differently.

The first issue was reproduced and tested unpacking a game and running it.

For the second issue, I measured the average time between sleeps in the Tamper Machine and it's around 83.7~83.9ms on Windows/Release, even with multiple cheats running, so the timing is now between spec and no significant drift from the `Sleep` function itself was observed.

I re-tested the cheats from my previous PR #1928, those being:

- 60fps mod in MHGU
- 99 lives / 999 seconds / start as penguin cheats in NSMBD
- 10 coins cheat in MK8D

### CODE REVIEW

**Review by AcK77 — APPROVED** [review]

**Comment by Caian** [comment]
> thank you
> i found an issues! cheating not work for this format
> [invincible]
> 040B0000 02430204 1E249000

Thanks, this is a known issue to be addressed in another PR.

**Comment by Myoko** [comment]
> > thank you
> > i found an issues! cheating not work for this format
> > [invincible]
> > 040B0000 02430204 1E249000
> 
> Thanks, this is a known issue to be addressed in another PR.

thats good news, thank you

File: Ryujinx.HLE/HOS/TamperMachine.cs [hunk 1]
```diff
@@ -13,6 +13,8 @@ namespace Ryujinx.HLE.HOS
 {
     public class TamperMachine
     {
+        private const int TAMPERMACHINE_SLEEP_MS = 1000 / 12;
```
Comment (Thog) [inline Ryujinx.HLE/HOS/TamperMachine.cs:16]:
```suggestion
        // 83ms
        private const int TamperMachineSleepMs = 1000 / 12;
```

File: Ryujinx.HLE/HOS/TamperMachine.cs [hunk 2]
```diff
@@ -76,7 +78,7 @@ private void TamperRunner()
                 if (sleepCounter == 0)
                 {
                     sleepCounter = _programs.Count;
-                    Thread.Sleep(1);
+                    Thread.Sleep(TAMPERMACHINE_SLEEP_MS);
```
Comment (Thog) [inline Ryujinx.HLE/HOS/TamperMachine.cs:81]:
```suggestion
                    Thread.Sleep(TamperMachineSleepMs);
```

**Review by Thog — APPROVED** [review]
I think the constant need to be in PascalCase but otherwise lgtm :+1:

**Comment by Caian** [comment]
Thanks @Thog, fixed.

**Pull request merged**